### PR TITLE
Calendars in listview

### PR DESCRIPTION
**Pull request by adamgyulavari**

### CODE REVIEW

File: app/src/main/java/com/greenfox/kalendaryo/MainActivity.java [hunk 1]
```diff
@@ -70,5 +111,38 @@ public void displayData() {
...
+                KalModel kalModel = response.body();
+
+                List<Kalendar> kalendars = kalModel.getItems();
+
+                for (Kalendar kalendar: kalendars) {
+                    adapter.add(kalendar);
+                }
+
+                listView.setAdapter(adapter);
+                adapter.clear();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/MainActivity.java]: clear before adding

File: app/src/main/java/com/greenfox/kalendaryo/models/Kalendar.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    String description;
+    String etag;
+    boolean hidden;
+    String id;
+    String kind;
+    String location;
+    String notificationSettings;
+    boolean primary;
+    boolean selected;
+    String summary;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/models/Kalendar.java]: id and summary is needed right now

File: app/src/main/java/com/greenfox/kalendaryo/models/KalModel.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
+package com.greenfox.kalendaryo.models;
+
+import java.util.List;
+
+/**
+ * Created by barba on 04/01/2018.
+ */
+
+public class KalModel {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/models/KalModel.java]: Kalendar -> KalendarsResponse

File: app/src/main/java/com/greenfox/kalendaryo/adapter/CalendarAdapter.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+        public CalendarAdapter(@NonNull Context context) {
+            super(context, 0);
+        }
+
+        @NonNull
+        @Override
+        public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+
+            convertView = LayoutInflater.from(getContext()).inflate(R.layout.calendar_name, parent, false);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/adapter/CalendarAdapter.java]: if(convertview == null)

**Review by nezihcihan — APPROVED** [review]
looks good.

**Pull request merged**